lib/auth_app_web/router.ex

```diff
@@ -69,17 +68,13 @@ defmodule AuthAppWeb.Router do
       live "/users/settings", UserLive.Settings, :edit
       live "/users/settings/confirm-email/:token", UserLive.Settings, :confirm_email
     end
+
+    post "/users/update-password", UserSessionController, :update_password
   end
 
   scope "/", AuthAppWeb do
     pipe_through [:browser]
 
     delete "/users/log-out", UserSessionController, :delete
-
-    live_session :current_user,
-      on_mount: [{AuthAppWeb.UserAuth, :mount_current_user}] do
```

Comment:
at the moment, we don't use `:mount_current_user` any more, but I didn't remove it yet. We can check for more things to remove later.